postgres/matviews/sms_error_and_users_docs_replication.sql

```diff
@@ -49,12 +56,31 @@ create materialized view app_monitoring_sms_error_states as (
                                 (doc::jsonb#>>'{date,current}')::bigint / 1000
                             )::double precision
                         )
-                    )::DATE
+                    )::date
                     ORDER BY doc::jsonb#>>'{date,current}'
-                ) rank
+            ) AS rank
             FROM monitoring_docs WHERE doctype = 'monitoring'
-        ) md
-        JOIN monitoring_urls mu ON md.url_id = mu.id
+        ) AS md
+        JOIN monitoring_urls AS mu ON md.url_id = mu.id
     WHERE md.rank = 1
-);
-GRANT SELECT ON app_monitoring_sms_error_states TO superset;
\ No newline at end of file
+)
+  SELECT
+    sms.partner_name,
+    sms.reported,
+    sms.due,
+    sms.muted,
+    sms.failed,
+    sms.delivered,
+    sms.scheduled,
+    COALESCE((doc #>> '{replication_limit, count}')::int, 0) AS "users past replication_limit",
+    COALESCE((doc #>> '{couchdb,medic,doc_count}')::int, 0) AS doc_count
```

Comment:
sorry i missed this - can we show this as the "change in docs per day"?

Reply:
Ok noted